helm/alfresco-process-infrastructure/requirements.yaml

```diff
@@ -9,29 +9,29 @@ dependencies:
     alias: alfresco-identity-service
   - name: alfresco-process-springboot-service
     alias: alfresco-modeling-service
-    version: 2.2.1
-    repository: https://kubernetes-charts.alfresco.com/stable
```

Comment:
was it stable because we forgot to put it back to stable after the release? Or other reason?
if just about post release, ignore my comment

Reply:
I was not changed after the release. maybe as there were no changes on the spring-boot chart in incubator version. But I think this should be changed after each release